sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -606,38 +601,16 @@ class ResolveSessionCatalog(
           ignoreIfExists,
           replace)
       } else {
-        val FunctionIdentifier(function, database) =
-          parseSessionCatalogFunctionIdentifier(nameParts, "CREATE FUNCTION")
-        CreateFunctionCommand(database, function, className, resources, isTemp, ignoreIfExists,
-          replace)
-      }
-  }
-
-  // TODO: move function related v2 statements to the new framework.
-  private def parseSessionCatalogFunctionIdentifier(
```

Comment:
Move this method to `LookupCatalog.CatalogAndFunctionIdentifier` and drop the sql param.

Reply:
This PR needs the change?